system-addon/lib/TelemetryFeed.jsm

```diff
@@ -119,7 +118,10 @@ this.TelemetryFeed = class TelemetryFeed {
       return;
     }
 
-    session.session_duration = Math.round(Components.utils.now() - session.start_time);
+    if (session.perf.visibility_event_rcvd_ts) {
+      session.session_duration = perfService.absNow() - session.perf.visibility_event_rcvd_ts;
```

Comment:
@ncloudioj Sure, that's easy enough to do.  Will push an update with that change momentarily.